### PR TITLE
[STK-376] Fix deleting CR should remove volumes

### PR DESCRIPTION
**Pull request by faizanahmad055**

### CODE REVIEW

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/konfigurator:PR-4-1``

File: pkg/kube/mounts/manager.go [hunk 1]
```diff
@@ -76,21 +76,21 @@ func (mm *MountManager) removeVolume() error {
 		}
 	}
 
-	return kubereflect.AssignValueTo(mm.Target, VolumesFieldPath, volumes)
+	return kubereflect.AssignValueTo(mm.Target, VolumesFieldPath, desiredVolumes)
 }
 
 func (mm *MountManager) removeVolumeMounts() error {
```
Comment (rasheedamir) [inline pkg/kube/mounts/manager.go]: in this function we don't log anything? seems to be an important function which should log something

**Comment by rasheedamir** [comment]
@faizanahmad055 as a good practice always add description when you open PR

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/konfigurator:PR-4-2``

File: pkg/kube/mounts/manager.go (hunk 1 above)
Comment (faizanahmad055) [reply pkg/kube/mounts/manager.go]: @waseem-h is adding logs in #3 to cover this.

**Pull request merged**